modular_zubbers/code/game/machinery/vending/multisec.dm

```diff
@@ -0,0 +1,84 @@
+/obj/machinery/vending/wardrobe/sec_wardrobe
+	name = "\improper MultiSec Outfitting Station"
+	desc = "A vending machine stocked with a multitude of security outfits, including standardized uniforms in multiple colors and general equipment. Now with more Multi!"
+	icon = 'modular_zubbers/code/game/machinery/vending/multisec.dmi'
+	icon_state = "multisec"
+	products = list() // Overriding this to the default to prevent any issues when building the product menu
+	product_categories = list(
```

Comment:
```suggestion
	icon_state = "multisec"
	light_mask = null
	products = list() // Overriding this to the default to prevent any issues when building the product menu
	product_categories = list(
```